### ISSUE TITLE
ISO 19115 Constraint information - allow supplementary detail

### ISSUE THREAD

**Issue by PeterParslow** [ISO-TC211/StandardsTracker#435]
When expressing a license or IPR constraint (and perhaps others) it would be helpful to be able to provide a link to further information - e.g. the licence itself.

But in order to allow this 'otherConstraints' information, useConstraints/accessConstraints has to be set to 'otherRestrictions'.

Proposal: allow a link (XML anchor or other) for additional information, independent of the RestrictionCode chosen.

**Comment by ejbleys** [comment]
a) It would be useful to extend useConstraint code option something like "licenceInfromation" 
     and constraint on otherConstraint be extended to that codeValue
       {otherConstraints: only documented if 
            accessConstraints or useConstraints = (“otherRestrictions” or "licenceInfromation"}
b) MD_LegalConstraints contains mco:reference: cit:CI_Citation [0..*] (by inheritance) 
        that would be an appropriate place to provide a citation and link to a license information
Note:
1) most licenses restrict usage (not access)
2) accessConstraints (drawn from MD_RestrictionCode) would usually be: restricted; private; confidential; in-confidence.
3) useConstraints & accessConstraints are [0..*] eg useConstraint both license and otherRestriction
      the latter invoking otherConstraint containing information about the licence location
4) any gco:CharacterString can be substituted with gcx:Anchor (assuming namespace has be assigned)

**Comment by PeterParslow** [comment]
For interest, the European Union has a controlled list of "reason" to provide as an anchor link from accessConstraints (currently using ISO 19115:2003 / ISO 19139:2007). These are the "allowable reasons" for (EU member government) data to limit public access: https://inspire.ec.europa.eu/metadata-codelist/LimitationsOnPublicAccess. There is a more helpful description behind each link:

(a) confidentiality of the proceedings of public authorites
(b) international relations, public security or national defence
(c) the course of justice
(d) confidentiality of commercial or industrial information
(e) intellectual property rights
(f) confidentality of personal data
(g) interests of a person who supplied the data on a voluntary basis
(h) protection of the environment

I'm not saying that should be adopted by ISO, but it suggests that the list of access constraint values may be longer than in MD_RestrictionCode.

**Comment by ejbleys** [comment]
Agreed that the MD_RestrictionCode could readily be expanded in the next iteration of ISO 19115-1. Once INSIRE fully transfers to ISO 19115-1/2 that will become even more important as the concepts being described would be better encoded in the MD_Releasability class and terms mentioned used in the disseminationConstraints (type=MD_RestrictionCode [0..*])

**Comment by PeterParslow** [comment]
See also #516
